README.md

```diff
@@ -1,22 +1,21 @@
-<h1>Discord Oxygen</h1>
```

Comment:
Why remove this?

Reply:
It makes sense to replace it with the logo instead, it's basically repeating the word twice.